cluster/plan.go

```diff
@@ -617,7 +619,9 @@ func (c *Cluster) BuildKubeletProcess(host *hosts.Host, serviceOptions v3.Kubern
 	if c.IsCRIDockerdEnabled() {
 		Env = append(Env,
 			// Enable running cri-dockerd
-			fmt.Sprintf("%s=%s", KubeletCRIDockerdNameEnv, "true"))
+			fmt.Sprintf("%s=%s", KubeletCRIDockerdNameEnv, "true"),
+			fmt.Sprintf("%s=%s", CRIDockerdStreamServerAddressEnv, c.CRIDockerdStreamServerAddress),
+			fmt.Sprintf("%s=%s", CRIDockerdStreamServerPortEnv, c.CRIDockerdStreamServerPort))
```

Comment:
Sorry for not catching this in the previous review, but this will cause all kubelets to restart since there'll be a change in node plan even when users haven't set a value. Could you add a check so that we add these env vars only when k8s version >= august patches?

Reply:
The good thing is rke-tools will still change the port to default values when kubelet restarts, so there'll be a partial fix for older k8s versions. But I'm concerned about force restarting kubelets without users triggering cluster upgrade.

Reply:
updated